src/main/kotlin/racingcar/model/InputValidator.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.model
+
+object InputValidator {
+
+    private const val MIN_NAME_LENGTH = 1
+    private const val MAX_NAME_LENGTH = 5
+
+    fun validateCarName(carNames: List<String>) {
+        carNames.forEach { name ->
+            require(name.length in MIN_NAME_LENGTH..MAX_NAME_LENGTH)
+            { "각 자동차의 이름의 길이는 1~5자 이어야 합니다." }
+        }
+    }
+
+    fun findDuplicateName(carNames: List<String>): Map<String, Set<Int>> {
+        val duplicateNameIndexes = mutableMapOf<String, MutableSet<Int>>()
+        carNames.forEachIndexed { index, value ->
+            duplicateNameIndexes.computeIfAbsent(value) { mutableSetOf() }.add(index)
+        }
+        return duplicateNameIndexes.filterValues { it.size > 1 }
+    }
```

Comment:
중복 처리 하신 로직 잘 봤습니다!
코틀린의 `withIndex()`와 ` groupBy()`를 활용해서 더 간결하게 표현할 수도 있을 것 같습니다!
```suggestion
    fun findDuplicateName(carNames: List<String>): Map<String, List<Int>> {
        return carNames.withIndex()
            .groupBy({ it.value }, { it.index })
            .filterValues { it.size > 1 }
    }
```

Reply:
오.. 이런 방법이... 깔끔하고 보기 좋네요.
감사합니다!

Reply:
Set으로 처리하신 부분과 관련된 메서드를 처음 봐서 충분히 깔끔하다고 생각했는데 BaekCCl님이 제안해주신 코드도 좋아 보이네요. 짱입니다.